p2p/dialer.go

```diff
@@ -75,31 +76,36 @@ func (m *dialer) Dial(ctx context.Context, consumerID identity.Identity, service
 		return nil, fmt.Errorf("could not exchange config: %w", err)
 	}
 
-	var remotePort, localPort int
-	var conn0 *net.UDPConn
-	var serviceConn *net.UDPConn
-	if len(config.peerPorts) == 1 {
-		localPort = config.localPorts[0]
-		remotePort = config.peerPorts[0]
+	if _, err := firewall.AllowIPAccess(config.peerPublicIP); err != nil {
```

Comment:
@zolia I didn't added cleanup of the rule here.
1. The same rule is added during connection, and will be cleaned up on disconnect.
2. All rules are cleaned on node shutdown.

Reply:
AllowIPAccess() could probably check if required IP is not already added to avoid duplications. But not needed with this PR imho.